app/decorators/user_decorator.rb

```diff
@@ -103,7 +103,7 @@ def qrcode(otp_secret_key)
     }
     url = user.provisioning_uri(nil, options)
     qrcode = RQRCode::QRCode.new(url)
-    qrcode.as_png(size: 280).to_data_url
+    qrcode.as_png(size: 240).to_data_url
```

Comment:
this decreases the padding in the image around the actual QR code, doesn't change the barcode size itself